src/components/LandingPage/LandingPageInterface.ts

```diff
@@ -0,0 +1,7 @@
+import CreateGameService from "./CreateGameService";
```

Comment:
I'm not familiar with the use of interfaces in JavaScript. Is there a reason we're doing it here?

Reply:
It was to get the state to work with Typescript and React, honestly not sure if this is a good way to do it, doesn't seem like it, but it was the only way to get state to wire up properly and also get Typescript to be happy.
We were playing around with a lot just to get the typescript errors to go away, so I'll take another look at this and see if we still need it. 

But long story short: typescript tries to add interfaces? Or rather tries to add some features of OO programming.